pixart.js

```diff
@@ -1 +1,81 @@
+$(document).ready(function() {
+  color = $('.brush').css('background');
+  prior = [color];
 
+  //create swatches
+  for (i = 1; i <= 3; i++) {
+    $('.brush').append('<div></div>');
+    $('div').eq(i + 1).attr('class', 'swatch');
+    $('div').eq(i + 1).attr('id', 'swatch' + i);
+  }
+
+  $('.swatch').css('height', '30px');
+  $('.swatch').css('width', '30px');
+  $('.swatch').css('margin', '5px 5px 5px 130px');
+
+  //create canvas
+  for(i=1; i<=8000; i++){
+    var divNum = i+4;
+    $('body').append('<div></div>');
+    $('div').eq(divNum).attr('class', 'square');
+  }
+
+  $('.square').css('height', '10px');
+  $('.square').css('width', '10px');
+  $('.square').css('margin', '0');
```

Comment:
Can also give .css an object of key value pairs so:
```js
$('.square').css({
  height: '10px',
  width: '10px',
  margin: '5px 5px 5px 130px'
})
```
would have the same effect